frontend/src/metabase/nav/components/AppBar/AppBar.styled.tsx

```diff
@@ -27,37 +24,36 @@ export const AppBarRoot = styled.header`
   }
 `;
 
+export const LogoLinkContainer = styled.div`
+  position: relative;
+  margin-left: ${space(2)};
+
+  ${breakpointMaxSmall} {
+    margin-left: ${space(1)};
+  }
+`;
+
 export const LogoLink = styled(Link)`
   cursor: pointer;
   display: flex;
   align-items: center;
   justify-content: center;
-  border-radius: 6px;
-  left: 0;
+  border-radius: 0.375rem;
   padding: ${space(1)} ${space(2)};
-  margin-left: ${space(2)};
-  position: absolute;
   transition: opacity 0.3s;
 
   &:hover {
     background-color: ${color("bg-light")};
   }
-
-  ${breakpointMaxSmall} {
-    margin-left: ${space(1)};
-  }
 `;
 
 export const SidebarButtonContainer = styled.div`
-  left: 23px;
-  opacity: 0;
   position: absolute;
-  top: 4px;
+  top: 50%;
```

Comment:
One thing to consider here is whitelabeled logos that are pretty wide.

This will move the the sidebar icon to the right, maybe more than we would like.

I imagine using a vertical logo in whitelabel will position the sidebar icon a bit much to the left.

Reply:
@gusaiani Fixed. Not it's positioned from the top left corner, should be as before.